Tests/OpenWeatherMap.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
```

Comment:
Try to split this test file to different: 
1) Authorization positive
2) Authorization negative

and put appropriate tests to this files